layouts/about/list.html

```diff
@@ -9,9 +9,9 @@ <h2>Our consultants</h2>
 {{ range .Pages }}
 <article class="bio">
 
-  {{ with .Resources.GetMatch "photo" -}}
-  <img src="{{ .RelPermalink }}" class="profilepic">
-  {{- end }}
+  {{ $original := .Resources.GetMatch "photo" }}
+  {{ $resized := $original.Resize "200x" }}
+  <img src="{{ $resized.RelPermalink }}" class="profilepic">
```

Comment:
FYI, you can do stuff like this to limit the variables' scope.
```suggestion
  {{ with (.Resources.GetMatch "photo").Resize "200x" -}}
  <img src="{{ .RelPermalink }}" class="profilepic">
  {{- end }}
```

Reply:
Hva man lærer!